less/Subscription.less

```diff
@@ -73,6 +73,27 @@
     cursor: pointer;
 }
 
+.subscription--unpause-icon {
+    display: block;
+    width: 30px;
+    height: 30px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url('../images/tim-pocket-watch.svg');
+    background-size: 20px;
+    cursor: pointer;
+    &:after {
+        display: block;
+        width: 30px;
+        height: 30px;
+        text-align: center;
+        font-size: 30px;
+        font-weight: lighter;
+        opacity: 0.7;
+        content: '✕';
+    }
+}
+
```

Comment:
@vapaj the icon is basically composed of the one use for pause subscription with an 'x' over it. Maybe we could get on from Steffen later on?

Reply:
Yeah, ask Steffen about it, he might have ideas.